Sources_v3/StreamChatUI/ChatChannelList/ChatChannelList.swift

```diff
@@ -0,0 +1,108 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+open class ChatChannelList<ExtraData: ExtraDataTypes>: UICollectionViewController
+    where ExtraData.Channel: NameAndImageProviding {
+    
+    // MARK: - Properties
+    
+    public let controller: _ChatChannelListController<ExtraData>
+    public let uiConfig: UIConfig<ExtraData>
+    public var didSelectChannel: (_ChatChannel<ExtraData>) -> Void = { _ in }
+    
+    // MARK: - Init
+
+    public init(
+        controller: _ChatChannelListController<ExtraData>,
+        uiConfig: UIConfig<ExtraData> = .default
+    ) {
+        self.controller = controller
+        self.uiConfig = uiConfig
+        
+        let layout = UICollectionViewFlowLayout()
+        layout.minimumInteritemSpacing = 0
+        layout.minimumLineSpacing = 0
+        
+        super.init(collectionViewLayout: layout)
+    }
+    
+    @available(*, unavailable)
+    public required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: - Life Cycle
+    
+    override open func viewDidLoad() {
+        super.viewDidLoad()
+        
+        collectionView.backgroundColor = .white
```

Comment:
-> UI appearance

Reply:
removed in 47b487dba9229c15cc2c2858b828f8e6cde80c65